packages/react-components/src/components/ParticipantItem.tsx

```diff
@@ -216,28 +244,53 @@ export const ParticipantItem = (props: ParticipantItemProps): JSX.Element => {
         className={mergeStyles({ width: `calc(100% - ${menuButtonContainerStyle.width})`, alignItems: 'center' })}
       >
         {avatar}
-        {me && <Text className={meTextStyle}>{isMeText}</Text>}
+        {me && <Text className={meTextStyle}>{strings.isMeText}</Text>}
         <Stack horizontal className={mergeStyles(infoContainerStyle)}>
           {onRenderIcon && onRenderIcon(props)}
         </Stack>
       </Stack>
-      {menuItems && menuItems.length > 0 && (
-        <>
-          {menuButton}
-          <ContextualMenu
-            items={menuItems}
-            hidden={menuHidden}
-            target={containerRef}
-            onItemClick={onDismissMenu}
-            onDismiss={onDismissMenu}
-            directionalHint={DirectionalHint.bottomRightEdge}
-            className={contextualMenuStyle}
-            calloutProps={{
-              preventDismissOnEvent
-            }}
-          />
-        </>
+
+      {/* When the participantStateString has a value, we don't show the menu  */}
+      {!me && participantStateString ? (
+        <Text>{participantStateString}</Text>
```

Comment:
do we have styles to handle truncation of long strings of text here?

Reply:
Not currently.

Reply:
Yes, going to bring this up in front of AlexP tomorrow.
Creating an item for this.
https://skype.visualstudio.com/SPOOL/_workitems/edit/2946830